home.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <section class="main-bg my-5">
+      <div class="container-md mt-5 mb-5">
+        <h1 class="h1 heading">"Hello! Sharing world"</h1>
+        <h1 class="display-1 heading-sub fw-600 text-uppercase">
+          "Jah Malet Fetari" International Chess Tournament 2023
+        </h1>
+        <p class="h4 sub-heading">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Iste fuga
+          nisi laborum nemo delectus iure.
+        </p>
+        <p class="display-4">2023.03.15(WED) ~ 17(FRI)</p>
+        <p class="h5">Ethiopia Science Museum; 4 Kilo, Addis Ababa, Ethiopia</p>
+      </div>
+    </section>
+    <section class="content" id="program">
+      <div class="container-md">
+        <h2 class="text-white text-center py-5">Main Program</h2>
+        <div class="row d-flex justify-content-center program-container"></div>
+        <h2
+          class="text-white text-center py-5 text-decoration-underline fs-5 text-uppercase"
+        >
+          See the whole program
+        </h2>
+      </div>
```

Comment:
- [ ] Kindly 😊 try the mobile version to stick to the Behance design as you see here in the mobile there is another style and content for this text, so please try to make it like the Behance 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225011886-cddb5de7-b5aa-49a8-a6c0-ee9fb8b42527.png)  |  ![image](https://user-images.githubusercontent.com/83360636/225011832-34f416ec-ea91-433b-aa18-28048b596173.png)   |